drivers/staging/vc04_services/bcm2835-audio/bcm2835-pcm.c

```diff
@@ -35,12 +35,12 @@ static const struct snd_pcm_hardware snd_bcm2835_playback_spdif_hw = {
 	SNDRV_PCM_RATE_88200 | SNDRV_PCM_RATE_96000 |
 	SNDRV_PCM_RATE_176400 | SNDRV_PCM_RATE_192000,
 	.rate_min = 44100,
-	.rate_max = 192000,
+	.rate_max = 48000,
```

Comment:
Is this correct? If so, don't you also need to change the .rates above?

Reply:
It is correct (reverting a9c1660) but you are right, the rates should change as well.

Reply:
IMO the HDMI hardware does support 192kHz. But the channel count is incorrect for HDMI separate cards created by the bcm2835_audio_simple_newpcm call.

Reply:
The snd_bcm2835_playback_spdif_hw is intended for non-pcm (passthough). I don't believe we usefully support anything beyond 2 channel 48kHz (for dts and ac3).